mailet/standard/src/test/java/org/apache/james/transport/matchers/AtMostTest.java

```diff
@@ -121,46 +112,64 @@ void shouldThrowWhenMatchersConfigNameAsSpace() {
     }
 
     @Nested
-    class MultiplesMatcherConfigurationTest {
-        private AtMost multipleMatchers;
+    class MultipleMatchersConfigurationTest {
+        private AtMost atMost3;
+        private AtMost atMost5;
 
         @BeforeEach
         void setup() throws MessagingException {
-            this.multipleMatchers = new AtMost();
-            multipleMatchers.init(
+            this.atMost3 = new AtMost();
+            atMost3.init(
                 FakeMatcherConfig.builder()
-                    .matcherName("MultipleMatchers")
-                    .condition("randomName:3")
-                    .condition("randomName:5")
+                    .matcherName("AtMost")
+                    .condition("AtMost:3")
+                    .build());
+
+            this.atMost5 = new AtMost();
+            atMost5.init(
+                FakeMatcherConfig.builder()
+                    .matcherName("AtMost")
+                    .condition("AtMost:5")
```

Comment:
You create two matchers with the same attribute name? I think we want to demonstrate we can have matchers with differente attribute names, thus having separate counter executions?

Reply:
we have 2 matcher and it run with own counter executions even with the same name.